### ISSUE TITLE
[TTNN Dialect] Add ttnn.zerosOp

### ISSUE THREAD

**Issue by svuckovicTT** [tenstorrent/tt-mlir#374]
Add `ttnn.zerosOp` in TTNN dialect to match TTNN lib API.

**Comment by staylorTT** [comment]
@svuckovicTT Is this issue still active? Do we need to put someone on this?

**Comment by staylorTT** [comment]
Could this be a good community issue ? Bug bounty?

**Comment by svuckovicTT** [comment]
> Could this be a good community issue ? Bug bounty?

Not really, oversight of managing would cost more than just implementing it.

**Comment by svuckovicTT** [comment]
Turns out it's actually needed for a scenario, I'll add it.
